test/jdk/java/foreign/TestScope.java

```diff
@@ -119,4 +146,14 @@ void testDerivedBufferScope(MemorySegment segment) {
         IntBuffer view = buffer.asIntBuffer();
         assertEquals(expectedScope, MemorySegment.ofBuffer(view).scope());
     }
+
+    private static final MemorySegment ZEROED_MEMORY = MemorySegment.ofArray(new byte[8102]);
```

Comment:
The nearest power of two is 8192 (2<sup>13</sup>)
```suggestion
    private static final MemorySegment ZEROED_MEMORY = MemorySegment.ofArray(new byte[8192]);
```

Reply:
Good catch. This was a typo. However, the test works as intended and the PR is already integrated.